### PR TITLE
Added GridNav to KConfig

### PR DESCRIPTION
**Pull request by goodsireric**
### Description of the feature or fix

Added the option to enable GridNav in LVGL (a useful tool when you have a keypad instead of a touchscreen).

### CODE REVIEW

**Comment by brgl** [comment]
I see this has been added upstream by commit `016099e09c3bccdf6b0dd5269c1ec4d00eb07b86` - can you cherry-pick this one instead for more consistent history?

File: Kconfig [hunk 1]
```diff
@@ -974,6 +974,18 @@ menu "LVGL configuration"
...
+
+        config LV_USE_GRIDNAV
+            bool "Enable grid navigation"
+            default n 
+
+        config LV_USE_FRAGMENT
+            bool "Enable lv_obj fragment"
+            default n 
+
+        config LV_USE_IMGFONT
```
Comment (brgl) [inline Kconfig:986]: This is not part of upstream commit, I think you added it accidentally when cherry-picking.

**Comment by goodsireric** [comment]
@brgl How about now? Thank you for reviewing this

**Review by brgl — APPROVED** [review]
Looks good now

**Pull request merged**